src/jdk.jdi/share/classes/com/sun/tools/jdi/ReferenceTypeImpl.java

```diff
@@ -182,8 +182,7 @@ public int compareTo(ReferenceType object) {
 
     public String signature() {
         if (signature == null) {
-            // Does not need synchronization, since worst-case
-            // static info is fetched twice
+            // Does not need synchronization. Worst case is static info is fetched twice
```

Comment:
Could you add dot at the end of all updated comments

Reply:
Ok.